src/template/agentPagesAgentBrief.php

```diff
@@ -92,6 +94,8 @@
 
 			</ul>
 
+			</div>
```

Comment:
@jonlipp We'll need this end `div` tag moved back so `.wolfnet_aoLinks` and `.wolfnet_aoContact` are siblings again.